components/director/internal/domain/formationassignment/repository_test.go

```diff
@@ -0,0 +1,327 @@
+package formationassignment_test
+
+import (
+	"database/sql/driver"
+	"regexp"
+	"testing"
+
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/kyma-incubator/compass/components/director/internal/domain/formationassignment"
+	"github.com/kyma-incubator/compass/components/director/internal/domain/formationassignment/automock"
+	"github.com/kyma-incubator/compass/components/director/internal/model"
+	"github.com/kyma-incubator/compass/components/director/internal/repo/testdb"
+	"github.com/kyma-incubator/compass/components/director/pkg/pagination"
+)
+
+func TestRepository_Create(t *testing.T) {
+	suite := testdb.RepoCreateTestSuite{
+		Name:       "Create Formation Assignment",
+		MethodName: "Create",
+		SQLQueryDetails: []testdb.SQLQueryDetails{
+			{
+				Query:       `^INSERT INTO public.formation_assignments \(.+\) VALUES \(.+\)$`,
+				Args:        []driver.Value{TestID, TestFormationID, TestTenantID, TestSource, TestSourceType, TestTarget, TestTargetType, TestState, TestConfigValueStr},
+				ValidResult: sqlmock.NewResult(-1, 1),
+				InvalidRowsProvider: func() []*sqlmock.Rows {
+					return []*sqlmock.Rows{sqlmock.NewRows(fixColumns)}
+				},
+			},
+		},
+		ConverterMockProvider: func() testdb.Mock {
+			return &automock.EntityConverter{}
+		},
+		RepoConstructorFunc:       formationassignment.NewRepository,
+		ModelEntity:               faModel,
+		DBEntity:                  faEntity,
+		NilModelEntity:            nilFormationAssignmentModel,
+		IsGlobal:                  true,
+		DisableConverterErrorTest: true,
+	}
+
+	suite.Run(t)
```

Comment:
In all test suites there is a comment like: `// This test suite is not suitable for entities with embedded tenant in them.`

This, however, is outdated and misleading. Can I ask you to clean up those comments in the test suites.